src/main/java/com/faforever/client/chat/ChatUserControl.java

```diff
@@ -141,4 +297,27 @@ private void setClanTag(String newValue) {
       clanLabel.setVisible(true);
     }
   }
+
+  String path = "themes/default/images/chat/";
```

Comment:
This breaks theming. Instead, use:

    String theme = preferencesService.getPreferences().getTheme();
    ....setImage(ThemeUtil.themeFile(theme, "images/chat/host.png"));

we don't need to create a `ClassPathResource`